### ISSUE TITLE
Error installing 3.8 with PostgreSQL

### ISSUE THREAD

**Issue by marcosgdf** [Dolibarr/dolibarr#5386]
# Bug

```
Request 610 : CREATE TRIGGER update_customer_modtime BEFORE UPDATE ON llx_advtargetemailing FOR EACH ROW EXECUTE PROCEDURE update_modified_column_tms();    Error de SQL DB_ERROR_NOSUCHTABLE ERROR: 42P01: relation "llx_advtargetemailing" does not exist LOCATION: RangeVarGetRelidExtended, namespace.c:418
Creación de funciones
Request 610 : CREATE TRIGGER update_customer_modtime BEFORE UPDATE ON llx_ecm_documents FOR EACH ROW EXECUTE PROCEDURE update_modified_column_date_u(); Error de SQL DB_ERROR_NOSUCHTABLE ERROR: 42P01: relation "llx_ecm_documents" does not exist LOCATION: RangeVarGetRelidExtended, namespace.c:418
Creación de funciones
Request 610 : CREATE TRIGGER update_customer_modtime BEFORE UPDATE ON llx_printer_ipp FOR EACH ROW EXECUTE PROCEDURE update_modified_column_tms();  Error de SQL DB_ERROR_NOSUCHTABLE ERROR: 42P01: relation "llx_printer_ipp" does not exist LOCATION: RangeVarGetRelidExtended, namespace.c:418
Creación de funciones
Request 610 : CREATE TRIGGER update_customer_modtime BEFORE UPDATE ON llx_time_basket FOR EACH ROW EXECUTE PROCEDURE update_modified_column_tms();  Error de SQL DB_ERROR_NOSUCHTABLE ERROR: 42P01: relation "llx_time_basket" does not exist LOCATION: RangeVarGetRelidExtended, namespace.c:418
Creación de funciones
Request 610 : CREATE TRIGGER update_customer_modtime BEFORE UPDATE ON llx_timebasket_counter FOR EACH ROW EXECUTE PROCEDURE update_modified_column_tms();   Error de SQL DB_ERROR_NOSUCHTABLE ERROR: 42P01: relation "llx_timebasket_counter" does not exist LOCATION: RangeVarGetRelidExtended, namespace.c:418
```

**Comment by marcosgdf** [comment]
Also happening in 3.9

**Comment by eldy** [comment]
Can' reproduce
Creation of trigger are inside file install/pgsql/functions/functions.sql
But inside this file i don't see the CREATE... you mention. Can you see them in your install/pgsql/functions/functions.sql file ?

**Comment by marcosgdf** [comment]
Closed as duplicate of #4756
